github/__init__.py

```diff
@@ -22,7 +22,7 @@
 DEFAULT_HOOK_EVENTS = ['create', 'pull_request', 'push']
 DEFAULT_HOOK_NAME = 'web' # see https://developer.github.com/v3/repos/hooks/
 DEFAULT_HOOK_CONTENT_TYPE = 'json'
-CONCOURSE_ID = 'concourse_id'
+HOOK_CONCOURSE_ID = 'concourse_id'
```

Comment:
how about CONCOURSE_ID_ATTRIB[UTE] (this is the attribute with which we construct an id with which we identify one concourse instance